### PR TITLE
p1-modal component

### PR DESCRIPTION
**Pull request by Naveenchandar**

### CODE REVIEW

**Review by Rohitprasad83 — APPROVED** [review]
Code looks good.

File: components/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                        style="width: 837px; height: 443px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin" id="iframe">
+                    </iframe>
+                </div>
+            </section>
+        </main>
+    </div>
+</body>
+<script>
+    const modal = document.getElementById("modal_container");
```
Comment (hiteshverma27) [inline components/modal/modal.html]: put it in separate file if possible

File: components/modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    }
+
+    // When the user clicks anywhere outside of the modal, close it
+    window.onclick = function (event) {
+        if (event.target == modal) {
+            modal.style.display = "none";
+        }
+    }
+</script>
+<script src="../../javascript/toggle.js"></script>
```
Comment (hiteshverma27) [inline components/modal/modal.html:122]: you can also use "defer" attribute with script tags and put script on top so file looks clean

**Review by hiteshverma27 — APPROVED** [review]
- left few suggestions
- code is easy to understand 
- looks good to merge

**Pull request merged**